analytics/testapp/ProjectSettings/ProjectSettings.asset

```diff
@@ -166,7 +166,7 @@ PlayerSettings:
   buildNumber:
     iOS: 0
   AndroidBundleVersionCode: 1
-  AndroidMinSdkVersion: 16
+  AndroidMinSdkVersion: 26
```

Comment:
I'm not a fan of this being updated.  We should figure out what the issue here is, because 16 should be fine, and bumping this up just hides the problem.

Reply:
Update the version for now to unblock the build process.
Shawn created a ticket for this problem, and VJ is working on it. Will revert the change once it problem solved.

Reply:
We have a ticket. And talked with Shawn and VJ today, we are working on it now.